contracts/core/RebalancingSetToken.sol

```diff
@@ -747,12 +713,18 @@ contract RebalancingSetToken is
         // Create combined token Array
         address[] memory oldComponents = currentSetInstance.getComponents();
         address[] memory newComponents = nextSetInstance.getComponents();
+        
```

Comment:
nit: I prefer to not have the space when newComponents was invoked so that it can be called in the next line. Also makes the comment orphaned